plugins/database/redshift/redshift_test.go

```diff
@@ -131,32 +134,46 @@ func TestPostgreSQL_CreateUser(t *testing.T) {
 		t.Fatalf("err: %s", err)
 	}
 
-	usernameConfig := dbplugin.UsernameConfig{
+	usernameConfig := dbplugin.UsernameMetadata{
 		DisplayName: "test",
 		RoleName:    "test",
 	}
 
 	// Test with no configured Creation Statement
-	_, _, err = db.CreateUser(context.Background(), dbplugin.Statements{}, usernameConfig, time.Now().Add(time.Minute))
+	_, err = db.NewUser(context.Background(), dbplugin.NewUserRequest{})
 	if err == nil {
 		t.Fatal("Expected error when no creation statement is provided")
 	}
 
-	statements := dbplugin.Statements{
-		Creation: []string{testRedshiftRole},
+	const password = "SuperSecurePa55w0rd!"
+	req := dbplugin.NewUserRequest{
+		UsernameConfig: usernameConfig,
+		Password:       password,
+		Statements: dbplugin.Statements{
+			Commands: []string{testRedshiftRole},
+		},
+		Expiration: time.Now().Add(5 * time.Minute),
 	}
 
-	username, password, err := db.CreateUser(context.Background(), statements, usernameConfig, time.Now().Add(time.Minute))
+	resp, err := db.NewUser(context.Background(), req)
```

Comment:
These calls should use the `dbtesting` package when the error is expected to be nil so it uses a common set of behaviors such as creating a context with timeout.

Reply:
The one place where I have not done this was for the `DeleteUser` test, where I found that 2s was always too short a timeout. I'd prefer not to have to set environment variables to override the default for the whole test suite, so just decided to avoid the helper in that one instance instead.

Reply:
That's completely reasonable.